cli/tests/integrations/test_marathon.py

```diff
@@ -30,6 +30,7 @@ def test_help():
     dcos marathon app show [--app-version=<app-version>] <app-id>
     dcos marathon app start [--force] <app-id> [<instances>]
     dcos marathon app stop [--force] <app-id>
+    dcos marathon app kill [--scale] [--host=<host>] <app-id>
     dcos marathon app update [--force] <app-id> [<properties>...]
```

Comment:
No tasks already is there: test_kill_stopped_app. I added a test for missing apps: test_kill_missing_app